.github/ISSUE_TEMPLATE/crash_report.yml

```diff
@@ -0,0 +1,42 @@
+name: Crash report
+description: Submit a crash report
+title: '[CRASH] <short description>'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to post your crash report!
+
+        Please notice:
+        - If a module was involved, please open an issue in the module's repo instead!
+        - If you're using docker on Apple M1, please make sure the image you're using was compiled for ARM!
+  - type: textarea
+    id: crash-report
+    attributes:
+      label: Crash report
+      description: Paste the complete crash log. Please include a few lines from the log preceding the crash report to provide some context.
+      render: shell
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        Please provide the following additional information below:
+        - OS distribution and version
+        - Steps to reproduce (if any)
+  - type: textarea
+    id: additional-information
+    attributes:
+      label: Additional information
+      description: OS version, steps to reproduce and other useful info.
+      render: shell
+    validations:
+      required: false
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info? (optional)
+      placeholder: ex. email@example.com
```

Comment:
What is this for? Why wouldn't we just respond on the issue. I think it comes across a bit weird.

Reply:
It provides one field for the crash report and one field for additional info. It just creates an issue. Try my test repo to see it in action.

Reply:
I meant this specific contact field. Do we need it? Wouldn't we just use the issue?

Reply:
Ah that. I've seen some cases where more info was needed but couldn't be shared openly.

Otoh, we shouldn't ask them to share contact details openly. I'll delete it.